### PR TITLE
Updated paths to binaries, example

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updates documentation and example

Summary of changes:
- updated paths to binaries
- Snap capitalized
- updated example

How to verify it:
- run example

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,17 +118,20 @@ Loaded Time: Tue, 01 Dec 2015 06:19:48 EST
...
-$ snapctl metric list
+$ $SNAP_PATH/snapctl metric list
 ```
 
 Or see available metrics only for OSDs:
 ```
-$ snapctl metric list | grep ceph/osd
+$ $SNAP_PATH/snapctl metric list | grep ceph/osd
+```
+Download desired processor and publisher plugins eg.
```
Comment (katarzyna-z) [inline README.md]: In this example processor plugin is not needed. I suggest removing 'processor'.

Comment (marcin-krolik) [inline README.md]: @katarzyna-z removed mentioning processor

**Comment by katarzyna-z** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -51,14 +51,14 @@ Build the plugin by running make within the cloned repo:
...
 ```
-This builds the plugin in `/build/rootfs/`
+This builds the plugin in `/build/${GOOS}/${GOARCH}`
 
 ### Configuration and Usage
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 * Ensure `$SNAP_PATH` is exported
-`export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
+`export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/linux/x86_64`
```
Comment (lmroz) [inline README.md]: This line is not consistent with `This builds the plugin in ```/build/${GOOS}/${GOARCH}```

**Pull request merged**